setup.py

```diff
@@ -42,6 +42,7 @@ def read(file):
     'isort>=4.2.5',
     'collective.checkdocs>=0.2',
     'Pygments>=2.2.0',  # required by checkdocs
+    'mypy==0.501',
```

Comment:
It's just the current version. But pinning it avoids breaking of the testsuite if mypy pushes a new release with additional checks. It's probably better to simply bump the version from time to time.